locations/storefinders/uberall.py

```diff
@@ -50,3 +52,26 @@ def parse(self, response, **kwargs):
 
     def parse_item(self, item, feature, **kwargs):
         yield item
+
+    def storefinder_exists(response: Response) -> bool | Request:
+        # Example: https://www.yves-rocher.it/trova-il-tuo-negozio#!
+        # <script src="https://uberall.com/assets/storeFinderWidget-v2.js"></script>
+        if response.xpath('//script[contains(@src, "https://uberall.com/assets/storeFinderWidget-v2.js")]').get():
+            return True
+
+        # Example: https://www.aldi.pl/informacje-dla-klienta/wyszukiwarka-sklepu.html
+        # Example: https://www.hypovereinsbank.de/hvb/kontaktwege/filiale
+        # TODO: Needs playwright to detect this properly in most cases
+        # Loads into the DOM:
+        # <script type="module" src="https://locator.uberall.com/locator-assets/store-finder-widget-bundle-v2-modern.js?b=My4xODQuMjU="></script>
+        if response.xpath(
+            '//script[contains(@src, "https://locator.uberall.com/locator-assets/store-finder-widget-bundle-v2-modern.js")]'
+        ).get():
+            return True
+
+        return False
+
+    def extract_spider_attributes(response: Response) -> dict | Request:
+        return {
```

Comment:
We should be returning `key` only.

Some examples:

https://www.yves-rocher.it/trova-il-tuo-negozio#! has the key provided as:

```<div id="store-finder-widget" data-key="HLGRPp968JZaR0D235dXJa5fMRPHuA"```

https://www.aldi.pl/informacje-dla-klienta/wyszukiwarka-sklepu.html has the key provided as:

```<div data-showbacklink="false" id="store-finder-widget" data-key="ALDINORDPL_3Lul0K0H0Vy47Jz0wGZlN30mchboaO"```